daemons/gptp/common/ieee1588port.cpp

```diff
@@ -1274,17 +1276,20 @@ void IEEE1588Port::becomeSlave( bool restart_syntonization ) {
 
 	port_state = PTP_SLAVE;
 
-	/* TODO : Seems this be active here
+	/* TODO : Should this be active for non-automotive profile uses
```

Comment:
Seeing that discussion continued on this I also checked the spec and concur that not starting the sync receipt timer until a sync is received and the slave knows what rate to expect is the right thing to do. I have gone ahead and removed the TODO comment I had added and removed the commented out code as well to reduce confusion in the future.